site/content/docs/5.0/utilities/interactions.md

```diff
@@ -18,10 +18,14 @@ Change the way in which the content is selected when the user interacts with it.
 
 ## Pointer events
 
-Bootstrap provides `pe-none` and `pe-auto` classes to prevent or add element interactions.
+Bootstrap provides `.pe-none` and `.pe-auto` classes to prevent or add element interactions.
 
 {{< example >}}
 <p><a href="#" class="pe-none">This link</a> can not be clicked.</p>
```

Comment:
Maybe better add `tabindex="-1"` to these links?

Reply:
good point, oops...was so concerned with callout that i actually missed it :)

Reply:
fixed